File: Extras/io/src/python/helloBasic.py
# Illustrates basic I/O using stdin and stdout. Just echo "Hello " (name) for
# each name in a file.
# Michael Albert 2/2021

import sys

for line in sys.stdin:
	print("Hello " + line.rstrip());

	# The rstrip is needed because each line is newline terminated so just
	# printing would result in blank lines between. Try it and see!

File: Extras/io/src/java/HelloBasic.java
package io;


import java.util.Scanner;

/**
 * Illustrates basic I/O using stdin and stdout. Just echo "Hello " (name) for
 * each name in a file.
 * 
 * @author MichaelAlbert
 */
public class HelloBasic {

   public static void main(String[] args) {
    Scanner in = new Scanner(System.in);
    while (in.hasNextLine()) {
      System.out.println("Hello " + in.nextLine());
    }
  }
  
}
